### PR TITLE
Feature/merge release to develop

### PR DESCRIPTION
**Pull request by anew**
title says it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
-==========================
-LoadDataToSnowflake Action Plugin
-==========================
+# LoadDataToSnowflake Action Plugin
+
+[![cm-available](https://cdap-users.herokuapp.com/assets/cm-available.svg)](https://docs.cask.co/cdap/current/en/integrations/cask-market.html)
```
Comment (CuriousVini) [inline README.md:3]: should this be `https://docs.cdap.io/cdap/current/en/integrations/cask-market.html`?

File: pom.xml [hunk 2]
```diff
@@ -18,7 +18,7 @@
   <groupId>co.cask.hydrator</groupId>
   <artifactId>loadtosnowflake-action-plugin</artifactId>
   <modelVersion>4.0.0</modelVersion>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>1.0.1</version>
```
Comment (CuriousVini) [inline pom.xml:21]: Should we add `SNAPSHOT` version on develop?

File: README.md (hunk 1 above)
Comment (anew) [reply README.md:3]: I am just merging existing commits back to develop, not making any changes like package renaming. I can fix this in the next PR.

File: pom.xml (hunk 2 above)
Comment (anew) [reply pom.xml:21]: Normally yes, bit the next PR will immediately bump it to 1.1.0, so its pointless in this case

**Review by CuriousVini — APPROVED** [review]

**Pull request merged**